packages/ag-charts-enterprise/src/features/crosshair/crosshairLabel.ts

```diff
@@ -17,7 +17,7 @@ export const defaultLabelCss = `
     font: 12px Verdana, sans-serif;
     overflow: hidden;
     white-space: nowrap;
-    z-index: 99999;
+    z-index: 99998; // Needs to be below tooltips!
```

Comment:
Are these children of the new `ag-charts-canvas` div? If so, we may wish to do some follow up work to reduce these values, as all values will be relative to that div

Reply:
Yup - further refinement needed, my intent with this change was just to get back to something working for all of the known use-cases.